client/reader/share/index.jsx

```diff
@@ -73,18 +78,52 @@ var ReaderShare = React.createClass( {
 
 	getDefaultProps() {
 		return {
-			position: 'top left',
+			position: 'top',
 			tagName: 'li'
 		};
 	},
 
+	componentWillUnmount() {
+		if ( this._closeHandle ) {
+			clearTimeout( this._closeHandle );
```

Comment:
Aside: I kinda wish `defer` returned a `cancel` function similar to `debounce` and `throttle`. As-is, the overlap with `setTimeout` is pretty confusing (i.e. why use `defer` instead of `setTimeout`, if cancelling is only possible via `clearTimeout`?)